package.json

```diff
@@ -22,20 +22,19 @@
     "pelias-logger": "^1.2.1",
     "pelias-model": "^7.1.0",
     "pelias-wof-admin-lookup": "^7.3.0",
+    "temp": "^0.9.1",
     "through2": "^3.0.0",
     "through2-filter": "^3.0.0",
     "through2-map": "^3.0.0",
-    "through2-sink": "^1.0.0",
-    "tmp": "^0.1.0"
+    "through2-sink": "^1.0.0"
   },
   "devDependencies": {
     "jshint": "^2.9.4",
     "precommit-hook": "^3.0.0",
     "proxyquire": "^2.0.0",
     "stream-mock": "^2.0.3",
     "tap-spec": "^5.0.0",
-    "tape": "^5.0.0",
-    "temp": "^0.9.0"
```

Comment:
LOL, I like how this was in here twice somehow :shrug:

Reply:
Nah it's just used to be a `tmp` used to be a runtime dependency and `temp` was a dev dependency.

So when I deleted `tmp` I had to promote `temp` to being a runtime dependency otherwise an `npm install --production` would fail to bring it in.

Reply:
Ah. Something that _has_ happened in the past is we use a module as a dev dependency, but later start using it is a regular dependency as well. In this case usually one of two things happens:
- We forget to add it to `dependencies` (bad)
- We add it to `dependencies` and forget to remove it from `devDependencies` (not really a big deal but not ideal)

So i figured that's what had happened here.